vendor/mooltipass/mcCombinations.js

```diff
@@ -962,6 +1005,19 @@ var extendedCombinations = {
                     currentForm.combination.fields.password = mpJQ('input[type=password]');
                     currentForm.combination.autoSubmit = true;
                 }
+                if ((currentForm.combination.fields.password) && (!currentForm.combination.fields.username)){
```

Comment:
why is this one needed? this looks like a major change to our core logic.

Reply:
This is not "change to our core logic"
This is more like the addition to main functionality 
If on second (password) stage the user name was not found, we made one more try to find it.
If main functionality worked correctly, this peice will not be executed.
I took this from processing of another special combination.
In this case this not necessary (spare functionality)
If you want - I can remove it

Reply:
remind me for which website this is needed? amazon?